src/workflow-requests.ts

```diff
@@ -79,7 +97,17 @@ export const triggerRouteFunction = async ({
     return ok("workflow-finished");
   } catch (error) {
     const error_ = error as Error;
-    if (!(error_ instanceof WorkflowAbort)) {
+    if (
+      error instanceof QstashError &&
+      error.message.includes("can not append to a a cancelled workflow")
```

Comment:
Same here. Message might be refactored but status code should be same. We can use BadRequest(400) here. You can print the message on the log for debugging purposes.